core/src/test/java/com/linecorp/armeria/MethodChainingTest.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria;
+
+import com.linecorp.armeria.internal.testing.AbstractMethodChainingTest;
+
+class MethodChainingTest extends AbstractMethodChainingTest {}
```

Comment:
Shouldn't we put this class to all modules?

Reply:
If the abstract class was okay, I tried to write more. 😉

Reply:
I think it's okay. 😄

Reply:
Thank you for the review. 😄

I added test code to each module.